x-pack/plugins/enterprise_search/public/applications/app_search/routes.ts

```diff
@@ -10,11 +10,15 @@ import { docLinks } from '../shared/doc_links';
 export const DOCS_PREFIX = docLinks.appSearchBase;
 
 export const ROOT_PATH = '/';
+export const NOT_FOUND_PATH = '/404';
 export const SETUP_GUIDE_PATH = '/setup_guide';
 export const LIBRARY_PATH = '/library';
 export const SETTINGS_PATH = '/settings/account';
 export const CREDENTIALS_PATH = '/credentials';
-export const ROLE_MAPPINGS_PATH = '#/role-mappings'; // This page seems to 404 if the # isn't included
+
+export const ROLE_MAPPINGS_PATH = '/role-mappings';
```

Comment:
Oh, this one's a change request though (I think, according to Kibana convention) - we should generally be snake_casing our routes
```suggestion
export const ROLE_MAPPINGS_PATH = '/role_mappings';
```